### ISSUE TITLE
Ascensions breaking the game

### ISSUE THREAD

**Issue by AyySorento** [prinzstani/CookieBot#105]
After the bot reincarnates, nothing happens. Cookies per second is at 0 and nothing is bought, basically freezing the game. With the upgrades purchased while reincarnating, the game should start on its own without having to click.

If I disable the bot and manually reincarnate, things start on their own with no problems. I can then re-enable the bot and let it play. The problem here is that the bot is constantly trying to reincarnate, so it's continually breaking and needing human intervention. 

When my game does reincarnate, I get a message saying "Cookie Bot is trying to get the true neverclick achievement. Please do not click cookies now." Maybe it has something to do with that achievement?

I can provide more details. Just let me know what is needed.

**Comment by Kikithecat12345** [comment]
_True Neverclick_ is a shadow achievement requiring you to get 1 million cookies without clicking the big cookie once. 

What the bot is doing when it's supposedly doing nothing is waiting for the game to create a golden cookie. This golden cookie will hopefully give cookies, and after maybe a few golden cookies we'll be able to buy a cursor. 

By manually reincarnating, you are disabling the "Born Fresh" challenge, which disables any Heavenly Upgrades. This is required for True Neverclick.

In other words, this is working as intended.

**Comment by Kikithecat12345** [comment]
_True Neverclick_ is a shadow achievement requiring you to get 1 million cookies without clicking the big cookie once. 

What the bot is doing when it's supposedly doing nothing is waiting for the game to create a golden cookie. This golden cookie will hopefully give cookies, and after maybe a few golden cookies we'll be able to buy a cursor. It's the only way to make cookies without any towers and without clicking the big cookie.

By manually reincarnating, you are disabling the "Born Fresh" challenge, which disables any Heavenly Upgrades. This is required for True Neverclick. The bot knows this, and decides to do other things this ascension. If you let it get this achievement, it'll never do this behavior again on your save file.

In other words, this is working as intended.

We should maybe look into adding options to disable shadow achievements, perhaps. It's been suggested before. (#59)

**Comment by AyySorento** [comment]
That makes perfect sense. Thank you for the quick explanation. 

There might be other cases too, but maybe all the bot needs is an additional notification or some type of progress bar to show it's still running. In the case of "True Neverclick", a notification such as "Waiting for golden cookie to appear" might be helpful.

Waiting to do the shadow achievements last could also be an idea. Just spitballing. 

I've been using [Cookie Monster](https://cookiemonsterteam.github.io/CookieMonster/) for years and that tool has the option to display a timer until the next golden cookie, reindeer, or other time-based event, above the bakery's name. Something similar for the bot could also help show that it's still working in the background. 

Just some thoughts. Not requesting anything specific really. Again, thanks for the quick details. :)

**Comment by prinzstani** [comment]
Actually, there is an extra notification, which is the message you mention in your description.

The reason to work with true neverclick early in the game is that it is very similar to neverclick and both can be achieved at the same time. However, I see that once neverclick is achieved, we could wait with true neverclick to later in the game.

Maybe this will be implemented sometime.

**Comment by Kikithecat12345** [comment]
> I've been using [Cookie Monster](https://cookiemonsterteam.github.io/CookieMonster/) for years and that tool has the option to display a timer until the next golden cookie, reindeer, or other time-based event, above the bakery's name. Something similar for the bot could also help show that it's still working in the background.

This is related to #96.
